server/apps/publish/publish_filters/publish_filter.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+from superdesk import get_resource_service
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+
+logger = logging.getLogger(__name__)
+
+
+class PublishFilterResource(Resource):
+    schema = {
+        'publish_filter': {
+            'type': 'list',
+            'schema': {
+                'type': 'list',
+                'schema': Resource.rel('filter_condition', True)
+            }
+        },
+        'name': {
+            'type': 'string',
+            'nullable': False,
+        }
+    }
+
+    additional_lookup = {
+        'url': 'regex("[\w,.:-]+")',
+        'field': 'name'
+    }
+
+    datasource = {'default_sort': [('_created', -1)]}
+    privileges = {'POST': 'publish_queue', 'PATCH': 'publish_queue'}
+
+
+class PublishFilterService(BaseService):
+    def on_create(self, docs):
+        pass
```

Comment:
Same comment as above - is this no-op override intentional or is it perhaps redundant?